src/components/DefineCompany/BusinessContactInfo.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <div v-if="!isEditing">
+      <div>Email Address: {{contact.email}}</div>
+      <div>Phone: {{contact.phone}}   ext:{{contact.phoneExtension}}</div>
+    </div>
+    <v-card flat class="business-contact-container">
+      <v-form v-model="formValid"  v-if="isEditing" ref="form">
+        <v-row>
+          <v-col cols="12">
+            <v-text-field
+              filled
+              label="Email Address"
+              req
+              persistent-hint
+              :rules="emailRules"
+              v-model="contact.email">
+            </v-text-field>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col cols="12">
+            <v-text-field
+              filled
+              label="Confirm Email Address"
+              req
+              persistent-hint
+              :error-messages="emailMustMatch()"
+              v-model="contact.confirmEmail">
+            </v-text-field>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col cols="12" md="6">
+            <v-text-field
+              filled
+              label="Phone Number"
+              persistent-hint
+              hint="Example: (555) 555-5555"
+              type="tel"
+              v-mask="['(###) ###-####']"
+              v-model="contact.phone"
+              :rules="phoneRules">
+            </v-text-field>
+          </v-col>
+          <v-col cols="3">
+            <v-text-field
+              filled label="Extension"
+              persistent-hint
+              :rules="extensionRules"
+              v-mask="'#####'"
+              v-model="contact.phoneExtension">
+            </v-text-field>
+          </v-col>
+        </v-row>
+      </v-form>
+      </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue, Prop, Watch, Emit } from 'vue-property-decorator'
+import { mask } from 'vue-the-mask'
+
+// Interfaces
+import { BusinessContactIF } from '@/interfaces'
+
+@Component({
+  directives: { mask }
+})
+export default class BusinessContactInfo extends Vue {
+  // Props
+ @Prop({
+   default: ():BusinessContactIF => ({
+     email: '',
+     confirmEmail: '',
+     phone: '',
+     phoneExtension: ''
+   })
+ })
+  private initialValue: BusinessContactIF
+
+  @Prop({ default: false })
+  private isEditing: boolean
+
+  @Prop({ default: false })
+  private showErrors: boolean
+
+  // Properties
+  private contact: BusinessContactIF = this.initialValue
+
+  private formValid : boolean = false
+
+  // Rules
+  private emailRules = [
+    (v: any) => !!v || 'Email address is required',
```

Comment:
I'm not 100% sure but i believe we know these values will be strings.  So possibly `(v: string)` might be more suitable?

Reply:
Yes. will change the type to string.

Reply:
👍